src/Config/Parser.php

```diff
@@ -130,6 +135,10 @@ private function parseIntegrationClasses( array $widget ) {
 	 * @return array
 	 */
 	public function normalize( array $partialConfig ) {
-		return isset( $partialConfig['value'] ) ? [ $partialConfig ] : $partialConfig;
+		$isAssocArray = function( array $array ) {
+			return count( array_filter( array_keys( $array ), 'is_string' ) ) > 0;
+		};
```

Comment:
@strategio I don't see the point of overcomplicating the code here with a function.

Can't you just set the value of `$isAssocArray` to `count( array_filter( array_keys( $$partialConfig ), 'is_string' ) )`?


```php
public function normalize( array $partialConfig ) {
	$isAssocArray = count( array_filter( array_keys( $partialConfig ), 'is_string' ) )
	return $isAssocArray ? [ $partialConfig ] : $partialConfig;
}		
```

Reply:
Right @andreasciamanna, it's overkill... simplified as suggested.